porthtml.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  
+  <link
+  href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&amp;display=swap"
+  rel="stylesheet">
+  <title>ibrahim Portfolio</title>
+ 
+ 
+</head>
+
+<body class="body1">
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li class="nav"><a href="#">my logo </a></li>
+        <li class="pac"><a href="#contact">Contact</a></li>
+        <li class="pac"><a href="#PORTFOLIO">Portfolio</a></li>
+        <li class="pac"><a href="#about">about</a></li>
+        <li id="ham">
+          <img src="Union.svg" alt="menu image" class="menu-image ham" />
+        </li>
+      </ul>
+    </nav>
+    <div id="nav-two" class="nav-two">
+      
+      <ul id="main-ls">
+        <li><a href="#project-list"></a>Contact</li>
+        <li><a href="#about">Portfolio</a></li>
+        <li><a href="#contact">ABOUT</a></li>
+      </ul>
+    </div>
+  </header>
+  <main class="container1">
+    <img src="rec.svg" alt="" width="20" height="20">
+    <img src="abc.svg" alt="" width="20" height="20">
+
+    <div class="con"></div>
+    <section class="sec1">
+      <h1 class="Portfolio">
+        I'M IBRAHIM <br />
+        GLAD TO SEE YOU!
+      </h1>
+      
+      <p class="my_description">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through <br />
+        some of my work and experience! If you like what you see and have a
+        project you need <br />
+        coded, don’t hestiate to contact me.
+      </p>
+      <div class="icons">
+        <h6 class="LET"><a href="#">LET’S CONNECT</a></h6>
+        
+        
+        <ul>
+          <li>
+            <a href="https://github.com/ibrahim777764"><img src="github.svg"  alt="github" /></a>
+          </li>
+          <li>
+            <a href="https://medium.com/"><img src="medium.svg" alt="medium   icon" /></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/Ibrahim66650696"><img src="twitter.svg" alt=" twitter icon icon" /></a>
+          </li>
+          
+        </ul>
+        <img src="abc.svg" alt="" width="20" height="20">
+
+        <img src="rec.svg" alt="" width="20" height="20">
+
+      </div>
+    </section>
+
+ 
+  
+</main>
```

Comment:
Please try to match Figma design as match as possible in the background, font size, line height, and social media icons
| Your design | Figma design |
| --- | --- |
| ![screencapture-raw-githack-ibrahim777764-my-portfoliom-feature-portfolio-porthtml-html-2022-02-11-11_29_46](https://user-images.githubusercontent.com/24703171/153568365-3d354c66-8fd5-497a-9d4e-5c97937676d8.png) | ![Template 2-Mobile-Main](https://user-images.githubusercontent.com/24703171/153568749-4b3cae09-637e-4ba1-bb52-390cc6356f17.png) |